gulpfile.js

```diff
@@ -0,0 +1,35 @@
+const browserSync = require('browser-sync').create()
+const gulp = require('gulp')
+const postcss = require('gulp-postcss')
+const cssnano = require('gulp-cssnano')
+
+exports.buildStyles = () => {
+    return gulp.src('./styles/main.css')
+      .pipe(postcss())
+      .pipe(cssnano())
+      .pipe(gulp.dest('./dest'))
+      .pipe(browserSync.stream())
+}
+
+exports.buildTemplates = () => {
+  return gulp.src('./templates/**/*.html')
+    .pipe(gulp.dest('./dest'))
+}
+
+exports.build = gulp.parallel([
+  exports.buildStyles,
+  exports.buildTemplates
+])
+
+exports.default = () => {
+  gulp.watch('./styles/**/*.css', gulp.parallel([exports.buildStyles]))
```

Comment:
If you clone the repo and just run `npm install && npm start` nothing show up because the files have not been built yet. Instead it would probably be best to have Gulp run the builtStyles and buildTemplates task initially.
```suggestion
  gulp.watch('./styles/**/*.css', {ignoreInitial: false}, gulp.parallel([exports.buildStyles]))
```